### DISCUSSION TITLE
WebUI on FreeBSD

### DISCUSSION THREAD

**Discussion by PrimozR** [openscopeproject/TrguiNG#78] [General]
Haven't tried it yet at all, but can the WebUI be used on FreeBSD in a TrueNAS Core jail?

In that regard, has anyone tested the WebUI when running Transmission as an app on TrueNAS SCALE and/or in a Docker Container? I'm guessing this should work as SCALE is Linux based, not BSD.

I know in the past there have been some issues running custom WebUIs on Free/TrueNAS due to the BSD base.

**Comment by qu1ck** [comment]
For some context: transmission web UI does not need any kind of server side scripting/execution environment, it's not like running a php website. It's just a bunch of static files that transmission serves over http, your browser does all the work and talks to transmission using the same rpc API that any other standalone app uses.

Given the above, any system that allows you to change the environment variables for the transmission process OR overwriting the original transmission web UI files with your own will be able to support custom web UIs. I don't know anything about TrueNAS SCALE but with docker you can do either of those things.
